<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-2VH7RQS" failures="1" tests="14" name="com.worthPoint.testLayer.VaultTest" time="815.863" errors="0" timestamp="2023-09-20T18:50:17 IST" skipped="0">
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultMyTagsHeaderFunctionality" time="16.917"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultAddNewTagFunctionality" time="29.723"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyUpdateTagNameFunctionality" time="26.113"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyDelateTagFunctionality" time="23.340"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultMyTagsModuleFunctionality" time="38.482"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultCollectionsHeaderFunctionality" time="15.794"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultCreateCollectionFunctionality" time="35.820"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultCollectionShareButtonFunctionality" time="28.564"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultUpdateCollectionNameFunctionality" time="37.931"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultDeleteCollectionFunctionality" time="27.676"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultCollectionsModuleFunctionality" time="76.083"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultCollectionFilterModuleFunctionality" time="122.713"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultMyVaultModuleFunctionality" time="200.503"/>
  <system-out/>
  <testcase classname="com.worthPoint.testLayer.VaultTest" name="verifyVaultAddVaultItemWithInsuranceCompanyAndStorageLocationModuleFunctionality" time="136.204">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.worthPoint.testLayer.VaultTest.verifyVaultAddVaultItemWithInsuranceCompanyAndStorageLocationModuleFunctionality(VaultTest.java:1871)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyVaultAddVaultItemWithInsuranceCompanyAndStorageLocationModuleFunctionality -->
  <system-out/>
</testsuite> <!-- com.worthPoint.testLayer.VaultTest -->
